libtock/tock.h

```diff
@@ -22,57 +22,57 @@ typedef void (subscribe_upcall)(int, int, int, void*);
 // There are multiple failure and success versions based on how many and the
 // size of any included values.
 typedef enum {
-  TOCK_SYSCALL_FAILURE             =   0,
-  TOCK_SYSCALL_FAILURE_U32         =   1,
-  TOCK_SYSCALL_FAILURE_U32_U32     =   2,
-  TOCK_SYSCALL_FAILURE_U64         =   3,
-  TOCK_SYSCALL_SUCCESS             = 128,
-  TOCK_SYSCALL_SUCCESS_U32         = 129,
-  TOCK_SYSCALL_SUCCESS_U32_U32     = 130,
-  TOCK_SYSCALL_SUCCESS_U64         = 131,
+  TOCK_SYSCALL_FAILURE =   0,
+  TOCK_SYSCALL_FAILURE_U32 =   1,
+  TOCK_SYSCALL_FAILURE_U32_U32 =   2,
+  TOCK_SYSCALL_FAILURE_U64 =   3,
+  TOCK_SYSCALL_SUCCESS = 128,
+  TOCK_SYSCALL_SUCCESS_U32 = 129,
+  TOCK_SYSCALL_SUCCESS_U32_U32 = 130,
+  TOCK_SYSCALL_SUCCESS_U64 = 131,
   TOCK_SYSCALL_SUCCESS_U32_U32_U32 = 132,
-  TOCK_SYSCALL_SUCCESS_U32_U64     = 133
```

Comment:
:(, this one (and below) feels like a regression

is there some kind of escape hatch comment from uncrustify we could apply here?